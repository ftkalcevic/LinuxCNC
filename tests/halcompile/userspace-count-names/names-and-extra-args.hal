loadusr -W userspace_count_names names=swim,swish slap slop
list pin
unloadusr userspace_count_names
waitusr userspace_count_names

loadusr -W userspace_count_names slam slalom names=swat,swap slap
list pin
unloadusr userspace_count_names
waitusr userspace_count_names

loadusr -W userspace_count_names slip names=sock,slack
list pin
unloadusr userspace_count_names
waitusr userspace_count_names

