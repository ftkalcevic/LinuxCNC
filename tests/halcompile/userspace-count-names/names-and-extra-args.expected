count=0
names[0]=swim
names[1]=swish
argc=3
argv[0]=userspace_count_names
argv[1]=slap
argv[2]=slop
argv[3]=(null)
swim.out
swish.out

count=0
names[0]=swat
names[1]=swap
argc=4
argv[0]=userspace_count_names
argv[1]=slam
argv[2]=slalom
argv[3]=slap
argv[4]=(null)
swap.out
swat.out

count=0
names[0]=sock
names[1]=slack
argc=2
argv[0]=userspace_count_names
argv[1]=slip
argv[2]=(null)
slack.out
sock.out

